spec/services/zendesk_cleaner_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe ZendeskCleaner do
+  let(:client) { double(ZendeskAPI::Client) }
+  let(:ticket1) { double(ZendeskAPI::Ticket, id: 1) }
+  let(:ticket2) { double(ZendeskAPI::Ticket, id: 2) }
+  let(:ticket3) { double(ZendeskAPI::Ticket, id: 3) }
+  let(:tickets) { [ticket1, ticket2, ticket3] }
```

Comment:
The Zendesk API search endpoint actually returns a ZendeskAPI::Collection - is it ok to just fake with an array?

Reply:
maybe you can use this
https://www.rubydoc.info/gems/zendesk_api/1.10.0/ZendeskAPI/Collection#%3C%3C-instance_method